Daemon/Peer.Daemon.Linux/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Peer.Server;
+
+namespace Peer.Daemon.Linux
+{
+    internal class Program
+    {
+        private const string _socketPath = "/var/run/peerless/peerless.sock";
+        static async Task Main()
+        {
+            Directory.CreateDirectory(Directory.GetParent(_socketPath)!.FullName);
+            File.Delete(_socketPath);
```

Comment:
Would this blow up an already running instance if a second one were started? I wonder if it would better to be aggressive about cleanup and fail to start if the socket is already open? I bet there's prior art about this somewhere.